adr/ADR-8.md

```diff
@@ -254,9 +260,13 @@ place a new message in the subject for the key with a nil body and the header `K
 This preserves history and communicate to watchers, caches and gets that a delete operation should be handled - clear cache,
 return no key error etc.
 
-#### Purging a bucket
+#### Purging a key
+
+Purge is like delete but history is not preserved. This is achieved by publishing a message in the same manner as Delete using the
+`KV-Operation: DEL` header but adding the header `Nats-Rollup: sub` in addition.
```

Comment:
Maybe make a note that this means do not simply do a straight subject purge, that was my original implementation which will be replaced

Reply:
That's what this is.  It says its like delete - ie. not a purge - and then expand what is different.

Note Purge has changed, the ADR was Purge() but its not Purge(key) since a bunch of people seemed to think strongly that Purge() isnt needed/wanted due to watches issues.  Purge(key) now is supported as of about 1 hour ago in the server, so this is new behavior, so not a surprise that yours dont match this - it was not possible.